### ISSUE TITLE
Additional Review Comments

### ISSUE THREAD

**Issue by imalsky** [kathlandgren/SWAMPE#6]
I've created a new issue as part of a JOSS review, I hope the comments are helpful. I will add to this thread shortly

**Comment by imalsky** [comment]
Overall the paper and code is a great addition to the field. Some notes that I believe could improve the work:

- Please expand the community guidelines for people who wish to contribute to SWAMPE or report issues
- I may be missing something, but can the references be rendered in the paper.md file? I see both the references file, as well as the paper file
- Remaking all the plots to the paper was easy thanks to the great documentation. However, in the quick start guide it would be helpful to specifically point the users to the jupyter notebooks available in the docs/source folder.
- Out of curiosity, are there plans to expand the model, for example an option to use a double-gray radiative transfer scheme instead of the stellar forcing here? Also, while the python code is wonderfully commented and will have an easier learning curve than FORTRAN, are any speedups possible through numba or something similar? Reviewing the code it looks like numpy is already used extensively for the data handling.

**Comment by kathlandgren** [comment]
Thank you for your comments and your enthusiasm! 

- The commuity guidelines for contributing to SWAMPE have been updated in the documentation and in the readme file. 

- The paper format follows the [JOSS example paper](https://joss.readthedocs.io/en/latest/submitting.html#example-paper-and-bibliography) format. The pdf is compiled using Github actions and the integrated pdf file is produced as a (zipped) artifact. The above link outlines this process, although I am uncertain whether the artifact is visible to other users. For your convenience, I am attaching the compiled draft of the paper here: [paper.pdf](https://github.com/kathlandgren/SWAMPE/files/10069849/paper.pdf).

- The links to the tutorials folder have been added at the top of the QuickStart and Plotting guides. 

- While there is no concrete timeline at this time, we are considering expanding the way forcing is handled at a future date, making it easier for users to input their own forcing functions (or, as you have suggested, to enter the parameters for a double-gray forcing scheme).

- During the development and testing phase, we have experimented with numba and math modules, and have concluded that relying on the scipy implementation of spherical harmonics leads to the best tradeoff between speed and stability.
